documentation/developers/status.md

```diff
@@ -107,10 +107,9 @@ Topics marked _in progress_ are already in the process of implementation by comm
 - [ ] Folder configuration (_in progress_)
   - [ ] [Reference](https://github.com/MiczFlor/RPi-Jukebox-RFID/wiki/MANUAL#manage-playout-behaviour)
   - [ ] Resume: Save and restore position (how interact with shuffle?)
-  - [ ] Single: Enable mpc single
-  - [ ] Shuffle: Enable mpc random (not shuffle)
-    - Rename to random, as this is mpc random
-  - [ ] Loop: Loop playlist
+  - [x] Repeat Playlist
```

Comment:
Doesn't this refere to the folder configuration feature rather then card actions?
Specific folder configuration is not yet implemented as far as i know.

Reply:
Agreed. But do we need this really for folders? Not sure if this super relevant

Reply:
Agreed. Done!